src/gui_common/dialogs/CustomDialog.cs

```diff
@@ -65,6 +96,8 @@ public class CustomDialog : CustomWindow
     private StyleBox closeButtonHighlight = null!;
 
     private Font? titleFont;
+
+    private List<AddWindowReorderingSupportToSiblings> windowReorderingNodes = new();
```

Comment:
This shouldn't be inside the pragma. Also this could probably be readonly so that this should be placed before the pragma section with godot-type holding variables.